### PR TITLE
iptorrents.py no password now just cookie

### PR DESCRIPTION
**Pull request by swordfish6975**
### Description of what this fixes:
fixes iptorrents.py - although they still send broken html so python2-lxml is required for this to work
also updated cat_ids

### Related issues:
...

### CODE REVIEW

**Comment by defhead** [comment]
So this fix #7271 wasn't enough?

**Comment by swordfish6975** [comment]
I believe it was at the time, was about a month ago that my updated version stopped working.

Thinking about it more overnight the username is no longer required either, but one extra ignored settings box never hurt anyone.

File: couchpotato/core/media/movie/providers/torrent/iptorrents.py [hunk 1]
```diff
@@ -11,9 +11,9 @@ class IPTorrents(MovieProvider, Base):
 
     cat_ids = [
         ([87], ['3d']),
-        ([89], ['bd50']),
-        ([48], ['720p', '1080p']),
-        ([101], ['2160p']),
+        ([89, 90], ['bd50']),
```
Comment (defhead) [inline couchpotato/core/media/movie/providers/torrent/iptorrents.py]: Why touching these? Does it have something to do with the login problem?

**Comment by defhead** [comment]
Just tried this fix, seems to be working great

File: couchpotato/core/media/movie/providers/torrent/iptorrents.py (hunk 1 above)
Comment (swordfish6975) [reply couchpotato/core/media/movie/providers/torrent/iptorrents.py]: with out this change movies with in the IPT Quality   

- Movie/BD-Rip
- Movie/MP4
- Movie/Web-DL
- Movie/x254

will never be found with the search, I had it with a certain movie during testing

Comment (k0sr) [reply couchpotato/core/media/movie/providers/torrent/iptorrents.py]: I get it for 90 and 20, but not for.
100 is not just a 2160p.
100 = x265 and it has movies with 720p and 1080p qualities.

Comment (defhead) [reply couchpotato/core/media/movie/providers/torrent/iptorrents.py]: I get it for 90 and 20, but not for.
100 is not just a 2160p.
100 = x265 and it has movies with 720p and 1080p qualities.

Comment (swordfish6975) [reply couchpotato/core/media/movie/providers/torrent/iptorrents.py]: Agreed, though this was the best mapping I could come up with, not sure what the perfect solution is here?

**Comment by jcak** [comment]
Working for me as well. Thanks for this fix

**Comment by generalsnipe** [comment]
Well, this explains why I got about 10 notifications in a row. Thanks for the fix.

**Comment by matthewcky2k** [comment]
@RuudBurger  can this get a merge into develop and master?

**Comment by matthewcky2k** [comment]
@RuudBurger ?

**Comment by defhead** [comment]
@RuudBurger Could u give us some news about this PR plz?

**Pull request merged**